flang/test/Lower/OpenMP/Todo/omp-declare-mapper.f90

```diff
@@ -21,7 +21,7 @@ subroutine declare_mapper_1
   type (my_type2)        :: t
   real                   :: x, y(nvals)
   !$omp declare mapper (my_type :: var) map (var, var%values (1:var%num_vals))
-!CHECK: not yet implemented: OpenMPDeclareMapperConstruct
+!CHECK: not yet implemented: lowering symbol to HLFIR
```

Comment:
I'm surprised to see this TODO come up. Please could you fix this before merging so that we can maintain a helpful error message for the user.

Reply:
This error is now from an unhandled form of map clause rather than declare mapper. As such, I believe it's out of scope for this PR.

I will however subsequently look into fixing it in a separate PR, hope that doesn't hold up this PR.